docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSampleWithOptions.cs

```diff
@@ -28,15 +28,15 @@ public static void Example()
             // Convert the List<DataPoint> to IDataView, a consumble format to ML.NET functions.
             var data = mlContext.Data.LoadFromEnumerable(samples);
 
-            var options = new ML.Trainers.RandomizedPrincipalComponentAnalyzer.Options()
+            var options = new ML.Trainers.RandomizedPcaAnomalyDetectionTrainer.Options()
```

Comment:
>RandomizedPcaAnomalyDetectionTrainer [](start = 42, length = 36)

can we leave  "trainer" out of the name?

Reply:
used acronym for Pca.  Kept suffix Trainer in the name of the class

---
In reply to: [264101278](https://github.com/dotnet/machinelearning/pull/2903#discussion_r264101278) [](ancestors = 264101278)